File: Course_01/C Language/Matrizes/cada_linha.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n;
    printf("Qual a ordem da matriz? ");
    scanf("%d", &n);

    int mat[n][n], maior;

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            printf("Elemento [%d,%d]", i, j);
            scanf("%d", &mat[i][j]);
        }
    }
    printf("MAIOR ELEMENTO DE CADA LINHA: \n");
    for (int i = 0; i < n; i++) {
        maior = mat[i][0];
        for (int j = 0; j < n; j++) {
            if (maior < mat[i][j]) {
                maior = mat[i][j];
            }
        }
        printf("%d\n", maior);
    }
    return 0;
}

File: Course_01/Python/Matrizes/cada_linha.py
m = int(input("Qual a ordem da matriz? "))
n = m

matriz = [[0 for x in range(n)] for x in range(m)]

for i in range(0, m):
    for j in range(0, n):
        matriz[i][j] = int(input(f"Elemento [{i},{j}]: "))
print("Maior elemento de cada linha:")
for i in range(0, m):
    maior = matriz[i][0]
    for j in range(0, n):
        if maior < matriz[i][j]:
            maior = matriz[i][j]
    print(maior)